backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
Consider renaming the buttons from 'Test' and 'Tell Joke' to more descriptive names, such as 'Research' and 'Generate Joke', respectively. This will improve user experience and clarity.